public/app/plugins/datasource/prometheus/configuration/PromSettings.tsx

```diff
@@ -75,11 +75,19 @@ export const PromSettings = (props: Props) => {
       </div>
       <h3 className="page-heading">Misc</h3>
       <div className="gf-form-group">
+        <div className="gf-form">
+          <Switch
+            label="Disable query editor metric/label name lookup"
```

Comment:
this is a bit long for a Switch label for the old form styles (might work after switch to new form style). Maybe just `Disable metric lookup` and rest in the tooltip?